examples/deploy_bundle.py

```diff
@@ -6,35 +6,41 @@
 3. Destroys the units and applications
 
 """
+from juju.controller import Controller
 from juju import loop
-from juju.model import Model
 
 
 async def main():
-    model = Model()
-    print('Connecting to model')
-    # Connect to current model with current user, per Juju CLI
-    await model.connect()
-
-    try:
-        print('Deploying bundle')
-        applications = await model.deploy(
-            'cs:~juju-qa/bundle/basic-0',
-            channel='beta',
-        )
-
-        print('Waiting for active')
-        await model.block_until(
-            lambda: all(unit.workload_status == 'active'
-                        for application in applications for unit in application.units))
-        print("Successfully deployed!")
-        print('Removing bundle')
-        for application in applications:
-            await application.remove()
-    finally:
-        print('Disconnecting from model')
-        await model.disconnect()
-        print("Success")
+    controller = Controller()
+    # connect to current controller with current user, per Juju CLI
+    await controller.connect()
+
+    bundles = [('cs:~juju-qa/bundle/basic-0', 'beta'), ('juju-qa-bundle-test', None)]
```

Comment:
This works, but I'd like to see it better organized. Instead of stuffing everything into main, I'd like to see something like:

```
async def deploy_bundle(name, risk):
    ...

async def main():
    ...
    # Deploy bundle
    await deploy_bundle('juju-qa-bundle-test', None)

    # Deploy legacy bundle
    await deploy_bundle('cs:...', 'beta')
```

It becomes clear after some inspection that this is what you're doing. But it's not clear at first glance. And I think that we should be careful to make the code in examples very, very clear.